.circleci/ds005_outputs.txt

```diff
@@ -56,6 +57,7 @@ fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-T1w_preproc.
 fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_AROMAnoiseICs.csv
 fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_confounds.tsv
 fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_MELODICmix.tsv
+fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-cifti_variant-space1_preproc.dtseries.nii
```

Comment:
Isn't there supposed to be a `json` file that describes `space1`?

Reply:
I was under the impression the space would be embedded within the file's metadata - I'm open to generating a `json` file but it seems a bit overkill for how few permutations we currently have

Reply:
@satra Can you clarify your intent here? I was under the impression that space variants were to require a defining metadata file.

From my perspective, if we're only depending on internal metadata, then `variant-X` provides no information beyond `space-cifti`, except in that one hopes that the naming convention is consistent across datasets.

> it seems a bit overkill for how few permutations we currently have

Sure, but we have to consider the possibility that someone else is producing files and doesn't use the same `space1` and `space2` conventions.